examples/text_to_speech_synthesizer/create_mar.sh

```diff
@@ -2,13 +2,28 @@
 set -euxo pipefail
 
 cd /tmp
-rm torchhub.zip
+if [ -f "torchhub.zip" ] 
+then
+   rm torchhub.zip
+fi
 wget https://github.com/nvidia/DeepLearningExamples/archive/torchhub.zip
```

Comment:
Why not just use `-O` to overwrite the file?
```suggestion
wget -O https://github.com/nvidia/DeepLearningExamples/archive/torchhub.zip torchhub.zip
```